xbmc/Application.cpp

```diff
@@ -2356,6 +2356,15 @@ bool CApplication::OnAction(const CAction &action)
       g_windowManager.ActivateWindow(WINDOW_MUSIC_PLAYLIST);
     return true;
   }
+  if (action.GetID() == ACTION_PLAYER_PROCESS_INFO)
+  {
+    if ((m_pPlayer->IsPausedPlayback() || m_pPlayer->IsPlaying()) &&
```

Comment:
I understood that <code>if (m_pPlayer->GetCurrentPlayer() == "VideoPlayer")</code> return <code>false</code> in case videoplayer is inactive (does not play something, is paused, rewinds, ff, ...). Is this correct?